src/app.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+const path = require('path');
+const { copyFile } = require('./copyFile');
+
+const cp = () => {
+  const [source, dest] = process.argv.slice(2);
+
+  if (!source || !dest) {
+    throw new Error(
+      'You have to pass source and dest files path'
+    );
+  }
+
+  const sourcePath = path.join(__dirname, source);
+  const destPath = path.join(__dirname, dest);
+
+  copyFile(sourcePath, destPath);
```

Comment:
There's no check whether source and destination paths are the same. Copying a file to the same location should be prevented according to the task description.

Reply:
not fixed